### PR TITLE
OSOE-755: Adding windows headless shell visual testing baseline image

### LINKED ISSUE #329: Fix the BasicVisualVerificationTests.VerifyNavbar sample (OSOE-755)
Because it started to fail [like this](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/7158480532) under Windows.

> [VisualVerificationBaselineImageNotFoundException in Lombiq.Tests.UI.Samples.Tests.BasicVisualVerificationTests.VerifyNavbar(browser: Chrome): test/Lombiq.UITestingToolbox/Lombiq.Tests.UI.Samples/Tests/BasicVisualVerificationTests.cs#L72](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/commit/93a6fda570b3ea6aa0c67cc983177e26be19f09d#annotation_16301364131)
Lombiq.Tests.UI.Exceptions.VisualVerificationBaselineImageNotFoundException: Baseline image file was not found under the path D:\a\Open-Source-Orchard-Core-Extensions\Open-Source-Orchard-Core-Extensions\test\Lombiq.UITestingToolbox\Lombiq.Tests.UI.Samples\Tests\BasicVisualVerificationTests_VerifyNavbar_By_ClassName[Contains]_-navbar-brand_Win32NT_chrome-headless-shell.png and maxRetryCount is set to 2, so it won't be generated.    at Lombiq.Tests.UI.Extensions.VisualVerificationUITestContextExtensions.AssertVisualVerificationApproved(UITestContext context, IWebElement element, Action`2 comparator, Nullable`1 regionOfInterest, Action`1 configurator) in D:\a\Open-Source-Orchard-Core-Extensions\Open-Source-Orchard-Core-Extensions\test\Lombiq.UITestingToolbox\Lombiq.Tests.UI\Extensions\VisualVerificationUITestContextExtensions.cs:line 400    at Lombiq.Tests.UI.Extensions.VisualVerificationUITestContextExtensions.AssertVisualVerificationApproved(UITestContext context, By elementSelector, Action`2 comparator, Nullable`1 regionOfInterest, Action`1 configurator) in D:\a\Open-Source-Orchard-Core-Extensions\Open-Source-Orchard-Core-Extensions\test\Lombiq.UITestingToolbox\Lombiq.Tests.UI\Extensions\VisualVerificationUITestContextExtensions.cs:line 309    at Lombiq.Tests.UI.Extensions.VisualVerificationUITestContextExtensions.AssertVisualVerificationApproved(UITestContext context, By elementSelector, Double pixelErrorPercentageThreshold, Nullable`1 regionOfInterest, Action`1 configurator) in D:\a\Open-Source-Orchard-Core-Extensions\Open-Source-Orchard-Core-Extensions\test\Lombiq.UITestingToolbox\Lombiq.Tests.UI\Extensions\VisualVerificationUITestContextExtensions.cs:line 141    at Lombiq.Tests.UI.Samples.Tests.BasicVisualVerificationTests.<>c.<VerifyNavbar>b__2_0(UITestContext context) in D:\a\Open-Source-Orchard-Core-Extensions\Open-Source-Orchard-Core-Extensions\test\Lombiq.UITestingToolbox\Lombiq.Tests.UI.Samples\Tests\BasicVisualVerificationTests.cs:line 72    at Lombiq.Tests.UI.Extensions.AsyncExtensions.<>c__DisplayClass0_0`1.<AsCompletedTask>b__0(T target) in D:\a\Open-Source-Orchard-Core-Extensions\Open-Source-Orchard-Core-Extensions\test\Lombiq.UITestingToolbox\Lombiq.Tests.UI\Extensions\AsyncExtensions.cs:line 11    at Lombiq.Tests.UI.Services.UITestExecutionSession`1.ExecuteAsync(Int32 retryCount, String dumpRootPath) in D:\a\Open-Source-Orchard-Core-Extensions\Open-Source-Orchard-Core-Extensions\test\Lombiq.UITestingToolbox\Lombiq.Tests.UI\Services\UITestExecutionSession.cs:line 114    at Lombiq.Tests.UI.Services.UITestExecutionSession`1.ExecuteAsync(Int32 retryCount, String dumpRootPath) in D:\a\Open-Source-Orchard-Core-Extensions\Open-Source-Orchard-Core-Extensions\test\Lombiq.UITestingToolbox\Lombiq.Tests.UI\Services\UITestExecutionSession.cs:line 138    at Lombiq.Tests.UI.Services.UITestExecutionSession`1.ExecuteAsync(Int32 retryCount, String dumpRootPath) in D:\a\Open-Source-Orchard-Core-Extensions\Open-Source-Orchard-Core-Extensions\test\Lombiq.UITestingToolbox\Lombiq.Tests.UI\Services\UITestExecutionSession.cs:line 151    at Lombiq.Tests.UI.Services.UITestExecutor.ExecuteOrchardCoreTestInnerAsync[TEntryPoint](UITestManifest testManifest, OrchardCoreUITestExecutorConfiguration configuration, String dumpRootPath) in D:\a\Open-Source-Orchard-Core-Extensions\Open-Source-Orchard-Core-Extensions\test\Lombiq.UITestingToolbox\Lombiq.Tests.UI\Services\UITestExecutor.cs:line 83    at Lombiq.Tests.UI.Services.UITestExecutor.ExecuteOrchardCoreTestInnerAsync[TEntryPoint](UITestManifest testManifest, OrchardCoreUITestExecutorConfiguration configuration, String dumpRootPath) in D:\a\Open-Source-Orchard-Core-Extensions\Open-Source-Orchard-Core-Extensions\test\Lombiq.UITestingToolbox\Lombiq.Tests.UI\Services\UITestExecutor.cs:line 83

It seems that this only affects GitHub-hosted runners, and not GitRunners ones, see [this more recent successful build](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/7167417845).

This does NOT seem like a [Windows max path length](https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry) limitation, since the resulting _D:\a\Open-Source-Orchard-Core-Extensions\Open-Source-Orchard-Core-Extensions\test\Lombiq.UITestingToolbox\Lombiq.Tests.UI.Samples\Tests\BasicVisualVerificationTests_VerifyNavbar_By_ClassName[Contains]_-navbar-brand_Win32NT_chrome-headless-shell.png_ path is 248 characters long.

[Jira issue](https://lombiq.atlassian.net/browse/OSOE-755)

### PR DESCRIPTION
**Pull request by wAsnk**
[OSOE-755](https://lombiq.atlassian.net/browse/OSOE-755)
Fixes #329

[OSOE-755]: https://lombiq.atlassian.net/browse/OSOE-755?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Lombiq.Tests.UI.Samples/Tests/BasicVisualVerificationTests_VerifyNavbar_By_ClassName[Contains]_-navbar-brand_Win32NT_chrome-headless-shell.png
Comment (Piedone) [inline Lombiq.Tests.UI.Samples/Tests/BasicVisualVerificationTests_VerifyNavbar_By_ClassName[Contains]_-navbar-brand_Win32NT_chrome-headless-shell.png:1]: The Edge image is also missing. Locally, the Chrome one works but I get this:

> Lombiq.Tests.UI.Exceptions.VisualVerificationBaselineImageNotFoundException : Baseline image file not found, thus it was created automatically under the path E:\ShortPath\OSOCE2\test\Lombiq.UITestingToolbox\Lombiq.Tests.UI.Samples\Tests\BasicVisualVerificationTests_VerifyNavbar_By_ClassName[Contains]_-navbar-brand_Win32NT_edge-headless-shell.png. Please set its "Build action" to "Embedded resource" if you want to deploy a self-contained (like a NuGet package) UI testing assembly. If you run the test again, this newly created verification file will be asserted against and the assertion will pass (unless the display of the app changed in the meantime).

**Review by Piedone — CHANGES_REQUESTED** [review]

File: Lombiq.Tests.UI.Samples/Tests/BasicVisualVerificationTests_VerifyNavbar_By_ClassName[Contains]_-navbar-brand_Win32NT_chrome-headless-shell.png
Comment (wAsnk) [reply Lombiq.Tests.UI.Samples/Tests/BasicVisualVerificationTests_VerifyNavbar_By_ClassName[Contains]_-navbar-brand_Win32NT_chrome-headless-shell.png:1]: How do you get this?

Comment (wAsnk) [reply Lombiq.Tests.UI.Samples/Tests/BasicVisualVerificationTests_VerifyNavbar_By_ClassName[Contains]_-navbar-brand_Win32NT_chrome-headless-shell.png:1]: And how do you repro locally the headless-shell? :D

Comment (wAsnk) [reply Lombiq.Tests.UI.Samples/Tests/BasicVisualVerificationTests_VerifyNavbar_By_ClassName[Contains]_-navbar-brand_Win32NT_chrome-headless-shell.png:1]: Also why was it working with the problematic windows-2022 run here? I have so many questions...

Comment (wAsnk) [reply Lombiq.Tests.UI.Samples/Tests/BasicVisualVerificationTests_VerifyNavbar_By_ClassName[Contains]_-navbar-brand_Win32NT_chrome-headless-shell.png:1]: Okay I could repro locally, but this is weird af.

Comment (Piedone) [reply Lombiq.Tests.UI.Samples/Tests/BasicVisualVerificationTests_VerifyNavbar_By_ClassName[Contains]_-navbar-brand_Win32NT_chrome-headless-shell.png:1]: I've just run it from VS, but I don't know.

**Pull request merged**